pyrightconfig.json

```diff
@@ -3,8 +3,7 @@
         "lib"
     ],
     "exclude": [
-        "lib/rucio/db",
-        "lib/rucio/tests",
```

Comment:
Do we want to run pyright on tests?

Reply:
It only runs inside "lib". See a couple of lines above